tests/php/test-fieldmanager-checkboxes-field.php

```diff
@@ -7,9 +7,33 @@
  * @group checkboxes
  */
 class Test_Fieldmanager_Checkboxes_Field extends WP_UnitTestCase {
-	public $post_id;
-	public $post;
-	public $custom_datasource;
+/**
+	 * The post ID of the test post.
+	 *
+	 * @var int
+	 */
+	private int $post_id = 0;
```

Comment:
Note that this changes visibility of the property from public to private, although it should be fine in a test case.